src/test/scala/no/ndla/conceptapi/service/WriteServiceTest.scala

```diff
@@ -87,6 +87,7 @@ class WriteServiceTest extends UnitSuite with TestEnvironment {
       ("en"),
       Some(updatedTitle),
       Some(updatedContent),
+      None,
```

Comment:
Kanskje utvide testen med å legge til nytt metaimage for forklaringa? Du har ingen andre tester som tester at du får lagt til metaimage til en forklaring.